packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -54,11 +55,26 @@ export class ConfigUtil {
   }
 
   public static async getDefaultUsernameOrAlias(): Promise<string | undefined> {
-    const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
-    const defaultUsernameOrAlias = configAggregator.getPropertyValue(
-      OrgConfigProperties.TARGET_ORG
-    );
-    return defaultUsernameOrAlias ? String(defaultUsernameOrAlias) : undefined;
+    try {
+      const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
+      const defaultUsernameOrAlias = configAggregator.getPropertyValue(
+        OrgConfigProperties.TARGET_ORG
+      );
+      if (defaultUsernameOrAlias === undefined) {
```

Comment:
Instead of explicitly looking for undefined, perhaps change it to:
```
if (!defaultUsernameOrAlias) {
  return undefined;
}
```

This will work if defaultUsernameOrAlias is undefined or if it is null.